tests/test.ts

```diff
@@ -580,35 +581,64 @@ describe('Test saveProject function from utils', () => {
 });
 
 describe('Test clasp apis functions', () => {
+  before(function() {
+    if (isPR !== 'false') {
+      this.skip();
+    }
+    setup();
+  });
   it('should list apis correctly', () => {
     const result = spawnSync(
       CLASP, ['apis', 'list'], { encoding: 'utf8' },
     );
     expect(result.status).to.equal(0);
-    expect(result.stdout).to.contain('abusiveexperiencereport   - abusiveexperiencereport:v1');
-    expect(result.stdout).to.contain('youtubereporting          - youtubereporting:v1');
+    expect(result.stdout).to.contain('# Currently enabled APIs:');
+    expect(result.stdout).to.contain('# List of available APIs:');
   });
-  it('should enable apis correctly', () => {
+  it('should ask for an API when trying to enable', () => {
     const result = spawnSync(
       CLASP, ['apis', 'enable'], { encoding: 'utf8' },
     );
+    expect(result.status).to.equal(1);
+    expect(result.stderr).to.contain('An API name is required.');
+  });
+  it('should enable sheets', () => {
+    const result = spawnSync(
+      CLASP, ['apis', 'enable', 'sheets'], { encoding: 'utf8' },
+    );
     expect(result.status).to.equal(0);
-    expect(result.stdout).to.contain('In development...');
+    expect(result.stdout).to.contain('Enabled sheets');
   });
-  it('should disable apis correctly', () => {
+  it('should give error message for non-existent API', () => {
```

Comment:
Right now it is just an unhandled promise rejection. Should update the code to pass this test.